### DISCUSSION TITLE
User Story

### DISCUSSION THREAD

**Discussion by krutarth4** [krutarth4/SHYFTX#25] [General]
To discuss on the user story

**Comment by krutarth4** [comment]
@vahidjb in the user story, do u think it is must to have time selection as for me it doesn't make that much sense to be this much specific

**Reply by vahidjb** [reply]
What do you mean by time selection?The specific pickup time is must have. Did you mean that?

**Reply by krutarth4** [reply]
yeah but it will be too hactic and will remove the flexibility for the truckers

**Comment by vahidjb** [comment]
One of the issue we want to solve is the guarantee that truck would be ready to pick up at the right time to reduce the water. So basically it is a must have feature..and it is necessary for communication and preparing for farmers..more over any logistics I worked with has time setting preferences.. What we can do is offer estimated date or 2-3 day priod
